.pipeline/package.json

```diff
@@ -16,7 +16,13 @@
   },
   "author": "",
   "license": "Apache-2.0",
+  "devDependencies": {
+    "@bcgov/pipeline-cli": "^1.0.1-0"
+  },
   "dependencies": {
-    "pipeline-cli": "git+https://github.com/bcdevops/pipeline-cli.git#f939cb2abff3d3ee6b07a0f11c5abb28de07d49d"
+    "@bcgov/gh-deploy": "^1.1.1",
```

Comment:
you have the gh-deploy script to create github deployments but it is not being used in your deploy script

Reply:
haha right. I'm basically taking the config from devhub, haven't clean up yet ;P